File: app/src/main/java/net/sunniwell/aidl/bean/CategoryBean.java
package net.sunniwell.aidl.bean;

import java.io.Serializable;

public class CategoryBean implements Serializable {
    private static final long serialVersionUID = 1;

    /* renamed from: id */
    int f299id;
    String title;
    int total;

    public int getId() {
        return this.f299id;
    }

    public void setId(int id) {
        this.f299id = id;
    }

    public String getTitle() {
        return this.title;
    }

    public void setTitle(String title2) {
        this.title = title2;
    }

    public int getTotal() {
        return this.total;
    }

    public void setTotal(int total2) {
        this.total = total2;
    }
}

File: app/src/main/java/com/eliving/tv/aidl/bean/CategoryBean.kt
package com.eliving.tv.aidl.bean

import java.io.Serializable

class CategoryBean : Serializable {
    /* renamed from: id */
    var id = 0
    var title: String? = null
    var total = 0

    companion object {
        private const val serialVersionUID: Long = 1
    }
}